content/resources/customer-experience-toolkit.md

```diff
@@ -102,22 +113,23 @@ Publish customer service standards and train all employees so they understand th
 **Train your employees**
```

Comment:
The sub-headings in the "culture" section don't match the formatting of the other sections, please check.